datadog_checks_dev/datadog_checks/dev/tooling/commands/validate/codeowners.py

```diff
@@ -58,6 +84,16 @@ def create_codeowners_map():
 def codeowners(ctx):
     """Validate that every integration has an entry in the `CODEOWNERS` file."""
 
+    failed_integrations = validate_logs_assets_codeowners()
+    if failed_integrations:
+        for integration in failed_integrations:
+            echo_failure(f"/{integration}/assets/logs/ is not owned by {LOGS_TEAM}")
+        abort()
```

Comment:
```suggestion
	has_failed = True
```

I would drop it so we can go through the rest of the validation after :)